cepgen-toolfile.spec

```diff
@@ -0,0 +1,33 @@
+### RPM external cepgen 1.0
```

Comment:
@forthommel , this should be `cepgen-toolfile` i.e. `### RPM external cepgen-toolfile 1.0`

Reply:
Hence the infinite loop... Thanks!